spec/controllers/group_assignment_invitations_controller_spec.rb

```diff
@@ -58,6 +58,17 @@
         expect(group_assignment.group_assignment_repos.count).to eql(0)
         expect(user.repo_accesses.count).to eql(0)
       end
+
+      it 'does not allow if group has maximum number of members' do
+        allow_any_instance_of(RepoAccess).to receive(:silently_remove_organization_member).and_return(true)
+        group = Group.create(title: 'The Group', grouping: grouping)
+        group_assignment.update(max_members: 1)
+        user_1 = GitHubFactory.create_classroom_student
+        group.repo_accesses << RepoAccess.create(user: user_1, organization: organization)
+
+        expect_any_instance_of(ApplicationController).to receive(:flash_and_redirect_back_with_message)
+        patch :accept_invitation, id: invitation.key, group: { id: group.id }
+      end
     end
```

Comment:
It would also be nice to have a test showing that user is added to the group if the max number has not been met